app/src/lib/editors/win32.ts

```diff
@@ -520,6 +570,22 @@ export async function getAvailableEditors(): Promise<
     })
   }
 
+  if (atomBetaPath) {
+    results.push({
+      editor: ExternalEditor.Atom,
```

Comment:
These labels should be distinct, otherwise it's not clear which one I should choose:

<img width="464" src="https://user-images.githubusercontent.com/359239/61805819-e79a4b00-ae0c-11e9-92ca-4bd9e96bdff8.png">


```suggestion
      editor: ExternalEditor.AtomBeta,
```